manifests/init.pp

```diff
@@ -29,6 +30,12 @@
   ensure_packages($package_name)
 
   # ensure that the kernel module is loaded
+  if $kmod_package_name {
+    package { $kmod_package_name:
+      ensure => 'present',
+      before => Exec['modprobe drbd'],
```

Comment:
@rstory I think using a `before` here is a little neater than conditionally having a `require` in the Exec.